sdk/openai/openai/README.md

```diff
@@ -124,17 +124,23 @@ The main concept to understand is [Completions][azure_openai_completions_docs].
 ```javascript
 const { OpenAIClient } = require("@azure/openai");
 
-const client = new OpenAIClient(
+async function main(){
+  const client = new OpenAIClient(
   "https://your-azure-openai-resource.com/",
   new AzureKeyCredential("your-azure-openai-resource-api-key"));
 
-const { choices } = await client.getCompletions(
-  "text-davinci-003", // assumes a matching model deployment or model name
-  ["Hello, world!"]);
+  const { choices } = await client.getCompletions(
+    "text-davinci-003", // assumes a matching model deployment or model name
+    ["Hello, world!"]);
 
-for (const choice of choices) {
-  console.log(choice.text);
+  for (const choice of choices) {
+    console.log(choice.text);
+  }
 }
+
+main().catch((err) => {
```

Comment:
The catch handling is not very interesting, I think we can leave it out for conciseness.